backend/dataall/modules/dashboards/aws/dashboard_quicksight_client.py

```diff
@@ -0,0 +1,270 @@
+import logging
+import re
+import os
+import ast
+
+from botocore.exceptions import ClientError
+
+from dataall.aws.handlers.parameter_store import ParameterStoreManager
+from dataall.aws.handlers.quicksight import QuicksightClient
+from dataall.aws.handlers.secrets_manager import SecretsManager
+from dataall.aws.handlers.sts import SessionHelper
+
+log = logging.getLogger(__name__)
+log.setLevel(logging.DEBUG)
+
+
+class DashboardQuicksightClient:
+    _DEFAULT_GROUP_NAME = QuicksightClient.DEFAULT_GROUP_NAME
+
+    def __init__(self, username, aws_account_id, region='eu-west-1'):
+        session = SessionHelper.remote_session(accountid=aws_account_id)
+        self._client = session.client('quicksight', region_name=region)
+        self._account_id = aws_account_id
+        self._region = region
+        self._username = username
+
+    def register_user_in_group(self, group_name, user_role='READER'):
+        QuicksightClient.create_quicksight_group(self._account_id, group_name)
+        exists = False
+        user = self._describe_user()
+
+        if user is not None:
+            exists = True
+
+        if exists:
```

Comment:
```suggestion
        if user is not None:
```
+ helper boolean could be removed